src/main/java/org/gridsuite/voltageinit/server/computation/service/CancelContext.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.voltageinit.server.computation.service;
+
+import lombok.Getter;
+import org.gridsuite.voltageinit.server.computation.utils.MessageUtils;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.messaging.support.MessageBuilder;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import static org.gridsuite.voltageinit.server.computation.service.NotificationService.*;
+
+/**
+ * @author Anis Touri <anis.touri at rte-france.com>
+ */
+@Getter
+public class CancelContext {
+
+    private final UUID resultUuid;
+
+    private final String receiver;
+
+    public CancelContext(UUID resultUuid, String receiver) {
+        this.resultUuid = Objects.requireNonNull(resultUuid);
+        this.receiver = Objects.requireNonNull(receiver);
+    }
```

Comment:
Can be a record if it's immutable.
```suggestion
public record CancelContext(UUID resultUuid, String receiver) {
    public CancelContext {
        Objects.requireNonNull(resultUuid, "resultUuid cannot be null");
        Objects.requireNonNull(receiver, "receiver cannot be null");
    }
```

Reply:
Agreed. ReportContext was turned into a ReportInfos record and I find it much better.